starter-core/src/main/java/io/micronaut/starter/feature/function/awslambda/template/awsLambdaFunctionRequestHandlerSpock.rocker.raw

```diff
@@ -0,0 +1,36 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+class FunctionRequestHandlerSpec extends Specification {
+
+    @@AutoCleanup
+    @@Shared
+    FunctionRequestHandler handler = new FunctionRequestHandler()
+
+    void "test Handler"() {
+        given:
+        APIGatewayProxyRequestEvent request = new APIGatewayProxyRequestEvent()
+        request.setHttpMethod("GET")
+        request.setPath("/")
```

Comment:
nit: could be:

```
        request.httpMethod = "GET"
        request.path = "/"
```

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1209/commits/851470e7ee8747c5aef117b3558762162ff73a96